vnext/Scripts/Microsoft.ReactNative.targets

```diff
@@ -8,7 +8,9 @@
         <Reference Include="$(MSBuildThisFileDirectory)..\..\lib\uap10.0\Microsoft.ReactNative.winmd">
             <Implementation>Microsoft.ReactNative.dll</Implementation>
         </Reference>
-    <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\Microsoft.ReactNative.dll" />
-    <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\Microsoft.ReactNative.pri" />
+    <ReferenceCopyLocalPaths Condition="${Configuration) == 'Debug'" Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\debug\Microsoft.ReactNative.dll" />
+    <ReferenceCopyLocalPaths Condition="${Configuration) == 'Debug'" Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\debug\Microsoft.ReactNative.pri" />
+    <ReferenceCopyLocalPaths Condition="${Configuration) != 'Debug'" Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\Microsoft.ReactNative.dll" />
```

Comment:
> != 'Debug' [](start = 56, length = 11)

It would be better to be explicit about == 'Release' here just in case if someone adds another configuration and they start to conflict here.

Reply:
Thats why I did != Debug. That way at least they will always get one of the versions.  If I did == Release, then it wouldn't build if they made up some other configuration.